src/algorithms/calorimetry/CalorimeterIslandCluster.cc

```diff
@@ -1,8 +1,8 @@
-// Copyright 2022, David Lawrence
-// Subject to the terms in the LICENSE file found in the top-level directory.
-//
-//  Sections Copyright (C) 2023 Chao Peng, Wouter Deconinck, Sylvester Joosten, Dmitry Kalinkin
-//  under SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2023 Chao Peng, Wouter Deconinck, Sylvester Joosten, Dmitry Kalinkin, David Lawrence
```

Comment:
```suggestion
// Copyright (C) 2022, 2023 Chao Peng, Wouter Deconinck, Sylvester Joosten, Dmitry Kalinkin, David Lawrence
```
Technically we need all years.

Reply:
I won't mark this for all files.

Reply:
There are no other files that miss year 2022